Sources/SwiftDocC/Infrastructure/Topic Graph/AutomaticCuration.swift

```diff
@@ -232,6 +232,7 @@ extension AutomaticCuration {
         .`typealias`,
         .`typeProperty`,
         .`typeMethod`,
+        .`typeSubscript`,
```

Comment:
It looks like `SymbolGraph.Symbol.KindIdentifier` is `CaseIterable` - should we add a test that just confirms we're covering all possible cases here?

Reply:
That's a really good idea! I'll write one up.

Reply:
I changed the existing `testAutomaticTopics` test, which revealed that a few symbol kinds were also not being curated correctly.